.github/workflows/ci.yml

```diff
@@ -25,7 +25,7 @@ jobs:
 
       - name: Install dependencies
         shell: bash -l {0}
-        run: conda install pytest locket numpy toolz pandas blosc pyzmq -c conda-forge
+        run: conda install pytest locket numpy toolz pandas blosc pyzmq pyarrow -c conda-forge
```

Comment:
I see there's already a `requirements.txt` in the project, with some of these included. Why don't we use that, and we may also need a separate `requirements-dev.txt` or `requirements-test.txt`.

Reply:
I agree that's not an unreasonable thing to do. That said, I'd prefer to keep this PR as minimal as we can.